src/components/heros/PortfolioPageHero.vue

```diff
@@ -42,9 +50,21 @@ const totalVeBalLabel = computed((): string =>
 
 const isLoadingTotalValue = computed((): boolean => isLoadingPools.value);
 
+const showVeBalBalanceTooltip = computed(() => {
+  if (!veBalSyncSupportedNetworks.includes(networkId.value)) {
+    return false;
+  }
+
+  if (networksSyncState.value[networkId.value] === NetworkSyncState.Synced) {
+    return false;
```

Comment:
Does the state go back to unsynced if they lock more veBAL again?

Reply:
yes